hashicorp/boundary-ui
# Icu 12060 move rose filterable dropdown to core

:tickets: [Jira ticket](https://hashicorp.atlassian.net/browse/ICU-12060)

## Description

The Rose::FilterableDropdown was moved into the core addon as just `Dropdown`. I added some tests as well to go along with the new component.



## Screenshots (if appropriate):

## How to Test

Build a binary from the llb-client-daemon branch and use that binary in your local dev environment. You will need to move that binary into the ui/desktop/electron-app/cli folder as well. Be sure to start the desktop client using this command: ENABLE_MIRAGE=false BYPASS_CLI_SETUP=true yarn start:desktop

The dropdown should work as it has in the past. It does have a maxHeight set now via the `@height` param on the dropdown. So if you have many projects, it should only grow so tall and show a scroll bar.





## Checklist:
